tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/passes/ConfigurationPassImpl.scala

```diff
@@ -45,36 +46,39 @@ class ConfigurationPassImpl @Inject()(val options: WriteablePassOptions,
     * @return true, if the pass was successful
     */
   override def execute(): Boolean = {
+    val inModule = tlaModule.get
```

Comment:
I initially read `inModule` as a predicate that would be true if we are in a module. But, iiuc, this is returning the current module (that we are in), so maybe

```suggestion
    val currentModule = tlaModule.get
```

would be clearer?